code-challenges.js

```diff
@@ -21,41 +21,103 @@ const fruit4 = "kiwi"
 
 // Pseudo code:
 
+// Declare a function and name it hotEnough
+// Use the given variables and declare variable for boilingPoint at 212 
+// Set up a conditional statement to compare given values to boiling point
+    // Use string interpolation in the output
+
 const temp1 = 42
 const temp2 = 350
 const temp3 = 212
 
+const boilingPoint = 212
+
+const  hotEnough= (fahrenheit) => {
+    if(fahrenheit < boilingPoint){
+        return `${fahrenheit} is below boiling point`
+    } else if (fahrenheit > boilingPoint){
+        return `${fahrenheit} is above boiling point`
+    } else if (fahrenheit === boilingPoint){
+        return `${fahrenheit} is at boiling point`
+    }
+}
+
+console.log(hotEnough(temp1)) // 42 is below boiling point
+console.log(hotEnough(temp2)) // 350 is above boiling point
+console.log(hotEnough(temp3)) // 212 is at boiling point
 
 // --------------------2) Create the code that will combine the two arrays and return the length using the variables provided below. Expected output: 10
 
 // Pseudo code:
 
+// Combine the arrays using the getter concat
+// Use the length built in method and console log to display combined length
+
 const myNumbers1 = [3, 7, 0, 36, -9]
 const myNumbers2 = [8, -7, 42, 9, 13]
 
+console.log(myNumbers1.concat(myNumbers2).length) // 10
 
 // --------------------3) Create the code that will reverse the letters of a string using the test variable provided below. Expected output: "2202 atleD"
 
 // Pseudo code:
 
+// Use split to take the string and make a newArray with single characters
+// Use the reverse and join built in method to combine new array back into a string
+    // Initially had reverse and join on separate lines of code but found it works on same line also
+
 const currentCohort = "Delta 2022"
 
+newArray = currentCohort.split("")
+
+console.log( newArray.reverse().join("")) // 2202 atleD 
+
 
 // --------------------4) Create a statement that will evaluate whether each number in an array is even or odd. Use the test variable provided below. Expected output: "odd" "even" "odd" "even" "odd" "even"
 
 // Pseudo code:
 
+// Use a for loop to check each value in the array
+    // Starting index of 0 and increments of 1 until last index in array
+    // Code logic into the loop
```

Comment:
In the future, please provide what kind of code logic will be placed in loop in your pseudocode.